package.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "wickrio-bot-api",
-	"version": "5.62.1",
+	"version": "5.62.2",
 	"description": "The official Wickr IO Bot API Framework",
 	"main": "src/WickrIOBot.js",
 	"dependencies": {
 		"fs": "0.0.2",
 		"simple-encryptor": "^3.0.0",
-		"wickrio_addon": "5.61.x"
+		"wickrio_addon": "5.60.3"
```

Comment:
please change to "5.60.x" so that if we make any bug fixes to that version it will be used.